src/Apps/Sell/Utils/previousSubmissionUtils.ts

```diff
@@ -0,0 +1,29 @@
+import { SellFlowStep } from "Apps/Sell/SellFlowContext"
+import { useEffect, useState } from "react"
+
+export const storePreviousSubmission = (
+  submissionID: string,
+  step: SellFlowStep
+) => {
+  localStorage.setItem("previousSubmissionID", submissionID)
+  localStorage.setItem("previousSubmissionStep", step)
+}
+
+export const usePreviousSubmission = () => {
+  const [submissionID, setSubmissionID] = useState<string | null>(null)
+  const [step, setStep] = useState<SellFlowStep | null>(null)
+
+  const getSubmission = async () => {
+    const id = await localStorage.getItem("submissionID")
+    const step = await localStorage.getItem("currentStep")
+
+    setSubmissionID(id as string)
+    setStep(step as SellFlowStep)
+  }
+
+  useEffect(() => {
+    getSubmission()
+  }, [])
```

Comment:
This effect should run in the browser, but it seems to only run on the server. @damassi @dzucconi, do you have any idea how I could force this effect to run in the browser to fetch the data from localStorage?

Reply:
I don't see how it could possibly run on the server from this? What makes you think it's running in the browser?